pkg/apis/aws/v1alpha1/awsfederatedaccountaccess_types.go

```diff
@@ -23,10 +23,10 @@ const (
 // AWSFederatedAccountAccessSpec defines the desired state of AWSFederatedAccountAccess
 // +k8s:openapi-gen=true
 type AWSFederatedAccountAccessSpec struct {
-	// CustomerAccountID holds the account number associated with the AWS account that will be used to access the associated role
+	// CustomerAccountID holds the external AWS account ID
 	CustomerAccountID string `json:"customerAccountID"`
```

Comment:
ExternalCustomerAwsAccountID? I still think that CustomerAccountID might be confusing?

Reply:
:+1: